showyourwork/workflow/rules/compile.smk

```diff
@@ -7,6 +7,9 @@ Runs the script :doc:`pdf` to generate the article ``ms.pdf``.
 from pathlib import Path
 from showyourwork import paths
 
+maybe_synctex = "--synctex" if config["synctex"] else ""
+joined_user_args = " ".join(config["user_args"])
```

Comment:
It's best practice to move these to the `params` section of the rule below. For example,

```python
params:
    maybe_synctex="--synctex" if config["synctex"] else "",
    user_args=config["user_args"]
```

should do the trick (you shoulndn't need to join them manually).

Reply:
Cool. Thanks!